src/main/java/icons/SonarLintIcons.java

```diff
@@ -20,6 +20,7 @@
 package icons;
 
 import com.intellij.openapi.util.IconLoader;
+import com.intellij.util.IconUtil;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.intellij.util.IconUtil'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)